xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/drawer.vm

```diff
@@ -99,7 +99,8 @@ aria-label="$escapetool.xml($services.localization.render('core.menu.drawer.labe
       $globalPart
     #end
   </ul>
-</nav>
+</dialog>
+<span role='presentation' class='drawer-overlay'></span>
```

Comment:
A dialog element has a native backdrop, so there should be no need anymore for an overlay. See [this discussion](https://stackoverflow.com/questions/25864259/how-to-close-the-new-html-dialog-tag-by-clicking-on-its-backdrop) how to handle click events on it.